app/controllers/split_checkout_controller.rb

```diff
@@ -0,0 +1,44 @@
+class SplitCheckoutController < CheckoutController
+  def edit
+    redirect_to_step if request.path == "/checkout"
+    super
+  end
+
+  def update
+    params_adapter = Checkout::FormDataAdapter.new(permitted_params, @order, spree_current_user)
```

Comment:
This `Checkout::FormDataAdapter` is pretty gnarly. I'd prefer if we avoid using it, that way when we're finished we can actually delete it...